os_net_config/impl_ifcfg.py

```diff
@@ -1984,6 +1989,12 @@ def apply(self, cleanup=False, activate=True, config_rules_dns=True):
                         )
                         self.ifdown(interface_name)
                         self.remove_config(ifcfg_file)
+            for nmconn_file in glob.iglob(nm_cleanup_pattern()):
+                interface_name = Path(nmconn_file).stem
+                cmd = ['nmcli', 'connection', 'delete', interface_name]
+                msg = "Removing nm connection %s" % interface_name
```

Comment:
Can we updae this log meesage as  similar like ifcfg message, here also removing connection and deleting nm connection file?
or 
This message is specific to nm connection removal and part of execute function, can we add above one more log for cleanup message to have similar like ifcfg cleanup?

Reply:
I don't see that format used anywhere in theimpl_ifcfg.py[1]. Is this a guideline specific to this project? If that is the reason you're blocking, I can update it though.

[1] https://github.com/os-net-config/os-net-config/blob/master/os_net_config/impl_ifcfg.py#L125

Reply:
We are converging on the same, so that the logs could be interpreted based on device.  We are yet to handle it completely.
Example: https://github.com/os-net-config/os-net-config/blob/master/os_net_config/impl_ifcfg.py#L785